src/ai/susi/server/api/aaa/PasswordChangeService.java

```diff
@@ -71,8 +71,8 @@ public ServiceResponse serviceImpl(Query post, HttpServletResponse response, Aut
             salt = authentication.getString("salt");
         } catch (Throwable e) {
             DAO.log("Invalid password try for user: " + identity.getName() + " from host: " + post.getClientHost() + " : password or salt missing in database");
-            result.put("message", "invalid credentials");
-            throw new APIException(422, "Invalid credentials");
+            result.put("message", "Invalid credentials. ");
+            throw new APIException(422, "Invalid credentials. ");
```

Comment:
remove "." from throw new APIException(422, "Invalid credentials. "); 
I think it is not required as it is an error message from the API. Users will only see the message object of the JSON response.